Locate rocm_agent_enumerator, clang++, extractkernel and clang-offload-bundler from system PATH rather than from /opt/rocm which is the location of ROCm used by upstream

Cancel running dpkg to check existence of hip

--- Tensile-rocm-4.1.0/Tensile/Common.py
+++ Tensile-rocm-4.1.0/Tensile/Common.py
@@ -155,7 +155,7 @@ globalParameters["PrintTensorRef"] = 0
 globalParameters["PrintIndexAssignments"] = 0      # Print the tensor index assignment info
 globalParameters["PrintTensorRef"] = 0          # Print reference tensor.  0x1=after init; 0x2=after copy-back; 0x3=both
 globalParameters["PrintWinnersOnly"] = False      # Only print the solutions which become the fastest
-globalParameters["PrintCodeCommands"] = False  # print the commands used to generate the code objects (asm,link,hip-clang, etc)
+globalParameters["PrintCodeCommands"] = True  # print the commands used to generate the code objects (asm,link,hip-clang, etc)
 
 # TODO - remove this when NewClient is mainstream
 globalParameters["OldClientSourceTmp"] = True      # Use an intermediate sourceTmp dir to detect file changes and minimize rebuilds on old client
